Formula/amazon-efs-utils.rb

```diff
@@ -180,6 +182,10 @@ def install
     inreplace buildpath/"dist/efs-utils.conf", "/var", libexec/"var"
     inreplace buildpath/"dist/amazon-efs-mount-watchdog.plist", "/var", libexec/"var"
 
+    if libexec.to_s.start_with?("/opt/homebrew")
+      inreplace buildpath/"dist/amazon-efs-mount-watchdog.plist", "/usr/local", "/opt/homebrew"
```

Comment:
Thought: it seems like we don't need to conditionally apply this edit and instead could be written to ensure that it matches across conventional installations of Homebrew (on different macOS versions) and also unconventional installations (ie: homebrew installed elsewhere on the system). This would cover the differences in vended AMIs for `mac1.metal` and `mac2.metal` - and any "exotic" images that could be in use.

```ruby
inreplace buildpath/"dist/amazon-efs-mount-watchdog.plist", "/usr/local", HOMEBREW_PREFIX
```

https://docs.brew.sh/Formula-Cookbook#variables-for-directory-locations